tutor/src/screens/scores-report/late-work.jsx

```diff
@@ -9,43 +9,35 @@ import omit from 'lodash/omit';
 import { AsyncButton } from 'shared';
 import TaskResult from '../../models/course/scores/task-result';
 
-import TH from '../../helpers/task';
-
 class LateWorkMessages {
 
   displayName = 'LateWork';
 
   constructor(task) {
     this.task = task;
-    this.isAccepted = this.task.is_late_work_accepted;
-    this.status = this.isAccepted ?
-      TH.hasAdditionalLateWork(this.task) ? 'additional' : 'accepted'
-      :
-      'pending';
   }
 
-  score() {
-    if (this.status === 'accepted') {
-      return (
-        TH.getHumanUnacceptedScore(this.task)
-      );
-    } else {
-      return (
-        TH.getHumanScoreWithLateWork(this.task)
-      );
+  @computed get status() {
+    if (this.task.is_late_work_accepted) {
+      return this.task.hasAdditionalLateWork ? 'additional' : 'accepted';
     }
+    return 'pending';
+  }
+
+  @computed get isAccepted() {
+    return Boolean(
+      this.task.is_late_work_accepted && !this.task.hasAdditionalLateWork
+    );
+  }
+
+  score() {
+    return this.isAccepted ?
+      this.task.humanUnacceptedScore : this.task.humanScoreWithLateWork;
   }
 
   progress() {
-    if (this.status === 'accepted') {
-      return (
-        TH.getHumanUnacceptedProgress(this.task)
-      );
-    } else {
-      return (
-        TH.getHumanProgressWithLateWork(this.task)
-      );
-    }
+    return this.isAccepted ?
+      this.task.humanUnacceptedProgress : this.task.humanProgressWithLateWork;
```

Comment:
Same as above, should use the same formula as completedPercent()

Reply:
Ignore the previous comment, since this is only used when accepting/rejecting late work.